cmd/pulsartest/cmd/watch.go

```diff
@@ -0,0 +1,59 @@
+package cmd
+
+import (
+	"github.com/spf13/cobra"
+
+	"github.com/G-Research/armada/internal/armada/configuration"
+	"github.com/G-Research/armada/internal/pulsartest"
+)
+
+func watchCmd() *cobra.Command {
+	a := &pulsartest.App{}
+
+	cmd := &cobra.Command{
+		Use:   "watch",
+		Short: "Watch for Pulsar events",
+		Long:  "Watch for Pulsar events",
+		Args:  cobra.ExactArgs(0),
+		PreRunE: func(cmd *cobra.Command, args []string) error {
+			url, err := cmd.Flags().GetString("url")
```

Comment:
I know all the err'ing is a golang-ism, but could this be hidden in a private getArgs(...) method to make the rest of this func read cleaner?

Reply:
I've applied a desiccant in commit 6396b0ce - the new `processCmdFlags()` func in `cmd/pulsartest/cmd/util.go` handles this processing for both `watch.go` and `submit.go`, and total net line-count should be down a bit.